### ISSUE TITLE
Rich workspaces: If there is no Readme.md, don’t show editor placeholder but move into "+" menu

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#33960]
Currently, every folder has the empty "Add notes, lists or links …" on the top. This leads to various issues such as:
- Always uses a lot of vertical space, for every single folder
- People can accidentally click into it
- People end up finding the feature annoying and disabling it completely even though they might find it useful on a folder-by-folder basis

Instead, we should only show it directly on top of the file list if a Readme.md is present. If not, we show an entry "Add rich workspace" at the bottom of the "+ New" menu. (This entry of course does not need to show if a Readme.md is present.)

Before | After (if a Readme.md is present, of course still show it directly)
-|-
![image](https://user-images.githubusercontent.com/925062/189092679-ee99ff79-0d60-4c7e-aa44-fc6f3b6c7912.png)|![image](https://user-images.githubusercontent.com/925062/189093430-0a54fcda-63ed-4f8a-b89d-86d5cfe98508.png)

cc @eneiluj @mejo- @max-nextcloud @juliushaertl 

As discussed @karlitschek

**Comment by max-nextcloud** [comment]
Maybe this issue should live in the text app?
It would solve https://github.com/nextcloud/text/issues/1171.

**Comment by ya-d** [comment]
Great proposal!! Would also fix nextcloud/text#2234 in a much, much nicer way.

Somewhat related: If you’re putting your hands on this feature, may I also suggest renaming "rich workspaces" to something more intuitive? I could think of "Add description", "Add instructions", "Add readme" or something similar. (And maybe move it more to the top of the menu?) @jancborchardt 

I don’t think anybody is doing any work in this small area. You’d much rather give a description of folder contents/structure, instructions on how to work with them, a checklist or whatever. Probably the Nextcloud itself could rather be called your (rich) workspace.

PS: In German its translated to "umfangreiche Arbeitsflächen" which really doesn’t make any sense to me. The term is probably hard to translate in general.

**Comment by jancborchardt** [comment]
@juliushaertl @max-nextcloud right, moving this to Office team project – feel free to move, but note that some part has to be done in the server.

@ya-d good point with the renaming – let’s name it to "Add description". :)

**Comment by jancborchardt** [comment]
Ah and with this change, let’s make sure to add a "Readme.md" to the root folder of https://github.com/nextcloud/example-files so the feature is still advertised.

**Comment by juliushaertl** [comment]
Makes a lot of sense. Moving to the text repo since, most of it should be doable in text.
